index.html

```diff
@@ -125,6 +125,11 @@ <h3 class="main-project-title">Multi-Post Stories</h3>
         </div>
         <div class="art-printing-wrapper">
           <div class="art-printing-text">
+            <h3 class="art-printing-title">Profesional Art Printing Data</h3>
+            <p class="art-printing-description">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+             </p>
```

Comment:
- [ ] **[OPTIONAL]** Great job on incorporating the previous reviewer's suggestions! I just have a quick suggestion **to make your mobile view even more professional and user-friendly**. It seems like there's an issue with the layout on mobile devices where all the content is pushed to the left side and something strange is happening with your body. Would you mind taking a look at it? 🎯

  ### Here's a screenshot:
  ![image](https://user-images.githubusercontent.com/107952724/236124119-cf2077d7-78cd-4bf6-abe1-e778ab471818.png)